app/version.py

```diff
@@ -0,0 +1,5 @@
+__version__ = "3.0.0"
```

Comment:
This would need to be accounted for in our release process.

Reply:
it'd be awesome if we had a `caldera` top-level package, but in absence of that, I applied the python package versioning conventions to this: https://packaging.python.org/guides/single-sourcing-package-version/